src/plugins/vis_type_table/public/to_ast.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { getVisSchemas, Vis } from '../../visualizations/public';
+import { buildExpression, buildExpressionFunction } from '../../expressions/public';
+import { TableVisExpressionFunctionDefinition } from './table_vis_fn';
+import { OpenSearchaggsExpressionFunctionDefinition } from '../../data/common/search/expressions';
+
+export const toExpressionAst = (vis: Vis, params: any) => {
```

Comment:
Can you write a unit test for this function too? You can reference `src/plugins/vis_type_metric/public/to_ast.test.ts` since they are quite similar

Reply:
Yes. I have a separate unit test issue and will create all the unit tests.
Here is the issue: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/2856.

Reply:
I will add a unit test for this to_ast.ts